boards/openlabs-kw41z-mini/doc.txt

```diff
@@ -3,20 +3,23 @@
 @ingroup     boards
 @brief       Support for openlabs-kw41z-mini
 
- ### General information
+### General information
 
 This is an open-source development board shipped by openlabs.co with
 source files located at https://openlabs.co/OSHW/kw41z-mini
 
 A driver for the radio transceiver is available in #12277 (802.15.4 only).
 
+@note A version of this board exists that comes with the `M41W8VT4` (256k ROM)
+instead of the `M41W9VT4` (512k ROM). The boards are otherwise identical.
+See @ref boards_openlabs-kw41z-mini-256kib if you happen to have this version.
 
 ### Programming Pinout
 
 Programming and debugging can be done with a Raspberry Pi (or equivalent)
 running OpenOCD.
 
-
+```
```

Comment:
Either also just indent them or use `~~~~~~~~~` fencing. ``` fencing does not work with Doxygen markdown.

Reply:
It worked in my local version of `doxygen` (1.8.13)…
Will change it.

Reply:
Maybe it was just with syntax highlighting, not sure. Let's stay on the save side (Doxygen-version-wise).

Reply:
`rg -F "\`\`\`"` in `boards/` shows quite a few results. 

e.g. [`common/slwstk6000b`](https://github.com/RIOT-OS/RIOT/blob/master/boards/common/slwstk6000b/doc.txt#L182) or [`cc1312-launchpad/doc.txt`](https://github.com/RIOT-OS/RIOT/blob/master/boards/cc1312-launchpad/doc.txt#L58)

Reply:
See https://github.com/RIOT-OS/RIOT/pull/13617#discussion_r391120201.